src/seedDashboard/seedDashboard.scss

```diff
@@ -278,9 +278,8 @@
           justify-self: flex-start;
         }
         .subdescription {
-          padding-left: 29px;
+          padding: 16px 0px;
```

Comment:
Best to make only the minimal change necessary.  Can you please just change padding-left?

Reply:
I needed not only to change the left padding but also add top padding to the `.subDescription` class, it makes the code more readable in my opinion when is shorter and in general more performant.

Reply:
The card didn't ask for a change in the top padding did it?